### PR TITLE
fix trace no data

### PR DESCRIPTION
**Pull request by candyleer**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.
the trace menu has no data because the data field `time_bucket` in es(agent insert in) is inconsistent with the data(web search) ,so i fix it
- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

**Comment by candyleer** [comment]
result
image

File: apm-collector/apm-collector-storage/collector-storage-es-provider/src/main/java/org/apache/skywalking/apm/collector/storage/es/dao/ui/SegmentDurationEsUIDAO.java [hunk 1]
```diff
@@ -57,8 +58,9 @@ public TraceBrief loadTop(long startSecondTimeBucket, long endSecondTimeBucket,
         List<QueryBuilder> mustQueryList = boolQueryBuilder.must();
 
         if (startSecondTimeBucket != 0 && endSecondTimeBucket != 0) {
-            //TODO second
-            mustQueryList.add(QueryBuilders.rangeQuery(SegmentDurationTable.COLUMN_TIME_BUCKET).gte(startSecondTimeBucket).lte(endSecondTimeBucket));
+            mustQueryList.add(QueryBuilders.rangeQuery(SegmentDurationTable.COLUMN_TIME_BUCKET)
+                    .gte(TimeBucketUtils.INSTANCE.secondToMinute(startSecondTimeBucket))
```
Comment (wu-sheng) [inline apm-collector/apm-collector-storage/collector-storage-es-provider/src/main/java/org/apache/skywalking/apm/collector/storage/es/dao/ui/SegmentDurationEsUIDAO.java]: This time format should already done in `TraceQuery#queryBasicTraces#L71`. And the query works in our demo env and product env.

**Comment by wu-sheng** [comment]
@candyleer Do you have this bug in 5.0.0-release or master code?

**Comment by candyleer** [comment]
@wu-sheng i found this in latest master branch

**Comment by peng-yongsheng** [comment]
This query DAO is not the point of the problem. This bug cause of my collector performance optimization.

**Comment by peng-yongsheng** [comment]
SegmentDurationSpanListener#parseFirst
```
- if (spanDecorator.getStartTimeMinuteTimeBucket() == 0) {
-            long startTimeMinuteTimeBucket = TimeBucketUtils.INSTANCE.getMinuteTimeBucket(spanDecorator.getStartTime());-            -spanDecorator.setStartTimeMinuteTimeBucket(startTimeMinuteTimeBucket);
-        }
-        timeBucket = spanDecorator.getStartTimeMinuteTimeBucket();
+ timeBucket = TimeBucketUtils.INSTANCE.getSecondTimeBucket(spanDecorator.getStartTime());
```

**Comment by candyleer** [comment]
ok .wait to be fixed

**Comment by peng-yongsheng** [comment]
@candyleer Can you help me? I'm coding other features.

**Comment by candyleer** [comment]
@peng-yongsheng  ok i'm glad to do that ,fix as you comment above

**Pull request merged**